framework/core/src/Api/Controller/ListNotificationsController.php

```diff
@@ -113,16 +113,16 @@ private function loadSubjectDiscussions(array $notifications)
         $ids = [];
 
         foreach ($notifications as $notification) {
-            if ($notification->subject && property_exists($notification->subject, 'discussion_id')) {
-                $ids[] = $notification->subject->discussion_id;
+            if ($notification->subject && $notification->subject->getAttribute('discussion_id')) {
```

Comment:
Could we use `&& ($discussionId = $notification->subject->getAttribute('discussion_id'))` to avoid repeating it?